src/models/dashboard-items.js

```diff
@@ -89,8 +99,16 @@ function getCharts(antigen, elements, organisationUnit, itemsMetadata, disaggreg
         .value();
 }
 
-function getTables(antigen, elements, organisationUnit, itemsMetadata, disaggregationMetadata) {
-    return _(dashboardItemsConfig.tables)
+function getTables(
+    antigen,
+    elements,
+    organisationUnit,
+    itemsMetadata,
+    disaggregationMetadata,
+    general = null
```

Comment:
It's a boolean,  right? then probably `general = false`. Also, having booleans as normal arguments is confusing when calling the function. I'd propse `..., disaggregationMetadata, {general = false} = {}`

Reply:
yes it's boolean of course, my bad. Applied suggested changes